clippy_lints/src/useless_conversion.rs

```diff
@@ -12,8 +12,8 @@ use rustc_session::{declare_tool_lint, impl_lint_pass};
 use rustc_span::sym;
 
 declare_clippy_lint! {
-    /// **What it does:** Checks for `Into`, `TryInto`, `From`, `TryFrom`,`IntoIter` calls
-    /// that useless converts to the same type as caller.
+    /// **What it does:** Checks for `Into`, `TryInto`, `From`, `TryFrom`, or `IntoIter` calls
+    /// that uselessly convert to the same type.
```

Comment:
```suggestion
    /// which uselessly convert to the same type.
```
I'd tend to prefer 'which' over 'that' :smile: